edit-post/index.js

```diff
@@ -68,6 +69,13 @@ export function initializeEditor( id, post, settings ) {
 
 	registerCoreBlocks();
 
+	dispatch( 'core/nux' ).triggerGuide( [
+		'core/editor.inserter',
+		'core/editor.settings',
+		'core/editor.preview',
+		'core/editor.publish',
+	] );
```

Comment:
I noticed that when dismissing a notice in a guide, and the next tip is not present in the DOM, no tip is being shown. Shouldn't we just skip to the next available tip?

Reply:
I don't think so. A tip in the guide might rely on the user having read a previous tip, e.g. _Now that you've learned about X, this is what Y does!_